scanner.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#written by rezshar
+#The easiest way to scan all local images
+./docker-index 2>&1 > /dev/null
+Val1=$(echo $?)
+   if [ "$?" -ne 0 ]
+   then
+      echo "Installing docker-index"
+      echo "please wait ..."
+      ./install.sh
+#    else
+#       echo "OK!"
+   fi
+
+if [ ! -d /var/lib/docker/image/overlay2/imagedb/content/sha256/ ]; then
```

Comment:
it seems that the docker-index would also work well with the short hash of `docker images -aq`
so you could just iterate over like this:

```
for i in $(docker images -qa); do docker-index cve --image "$i" DSA-2022-0001; done
```

assuming you have docker-index in your PATH.